core/src/main/scala/cats/data/Xor.scala

```diff
@@ -83,6 +83,19 @@ sealed abstract class Xor[+A, +B] extends Product with Serializable {
   def to[F[_], BB >: B](implicit F: Alternative[F]): F[BB] =
     fold(_ => F.empty, F.pure)
 
+  /**
+   * Transform this `Xor` into a [[XorT]] while lifting it into the specified Applicative.
+   *
+   * {{{
+   * scala> import cats.implicits._
+   * scala> val x: Xor[String, Int] = Xor.right(3)
+   * scala> x.toXorT[Option, String, Int]
+   * res0: cats.data.XorT[Option, String, Int] = XorT(Some(Right(3)))
+   * }}}
+   */
+  def toXorT[F[_], L, R](implicit F: Applicative[F], evL: A <:< L, evR: B <:< R): XorT[F, L, R] =
+    XorT(F.pure(bimap(evL, evR)))
```

Comment:
Hmmm if `toXorT` has to respecify all type parameters I am not sure I will favor it over `fromXor`